connectivity/netsocket/tests/UNITTESTS/netsocket/UDPSocket/CMakeLists.txt

```diff
@@ -28,8 +28,10 @@ target_sources(${TEST_NAME}
 target_link_libraries(${TEST_NAME}
     PRIVATE
         mbed-headers
+        mbed-headers-netsocket
```

Comment:
this is not included in mbed-headers?

Reply:
We will fix this part of https://github.com/ARMmbed/mbed-os/issues/14879 issue